Kudu.Core/Deployment/DeploymentManager.cs

```diff
@@ -544,6 +544,31 @@ private DeployResult GetResult(string id, string activeDeploymentId, bool isDepl
                 string repositoryRoot = _environment.RepositoryPath;
                 var perDeploymentSettings = DeploymentSettingsManager.BuildPerDeploymentSettingsManager(repositoryRoot, _settings);
 
+                string delayMaxInStr = perDeploymentSettings.GetValue(SettingsKeys.MaxRandomDelayInSec, false);
+                if (delayMaxInStr != null) {
+                    int maxDelay;
+                    if(!Int32.TryParse(delayMaxInStr, out maxDelay) || maxDelay < 0)
```

Comment:
I'd say it's fine to ignore the setting if it has an incorrect value. It's an advanced setting, and it's not that hard to get it right. I believe that's what we do with other similar settings.